src/discordChannelMonitor/DiscordChannelMonitor.js

```diff
@@ -0,0 +1,58 @@
+class DiscordChannelMonitor {
+  /**
+   * @param {String} channel - Channel that is monitored
+   * @param {String} responseStr - Message to reply to user when not a showcase.
```

Comment:
I'm wondering what the least intrusive way is to do this. If we want to keep the channel clean it might be better to not send a message in the channel itself. We could either send a DM (which might be blocked?) or we could send a block message and clean it up after a timer. What do you think?